### PR TITLE
Fixes #299 - Add and use a "get the platform version" algorithm

### LINKED ISSUE #299: `Sec-CH-UA-Platform-Version` header and `platformVersion` in the steps of `getHighEntropyValues()` doesn't point to the same text
Hi,

I found those two descriptions are not referencing to the same thing when I crawl the spec.
- [getHighEntropyValues](https://wicg.github.io/ua-client-hints/#getHighEntropyValues) looks up [user-agent-platform-version](https://wicg.github.io/ua-client-hints/#user-agent-platform-version) and it says `The user agent's operating system’s version. (e.g., "NT 6.0", "15", or "17G")`
- [sec-ch-ua-platform-version](https://wicg.github.io/ua-client-hints/#sec-ch-ua-platform-version) looks up its own logic.

They sounds slightly different according to the example, but I'm guessing they are expected to be the same.
Is my understanding correct?

Thanks!

### PR DESCRIPTION
**Pull request by miketaylr**
Fixes #299

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/ua-client-hints/pull/304.html" title="Last updated on May 25, 2022, 5:37 PM UTC (7bb7031)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/ua-client-hints/304/1b8a8b4...miketaylr:7bb7031.html" title="Last updated on May 25, 2022, 5:37 PM UTC (7bb7031)">Diff</a>

### CODE REVIEW

**Comment by miketaylr** [comment]
This should be a purely editorial change, in theory. PTAL @yoavweiss

File: index.bs [hunk 1]
```diff
@@ -644,49 +644,44 @@ The 'Sec-CH-UA-Platform-Version' Header Field {#sec-ch-ua-platform-version}
...
-[=Structured Header=] whose value MUST be a [=structured header/string=]
-[[!RFC8941]].
+information about the [=user agent/platform version=] on which a given [=user agent=] is executing.
+It is a [=Structured Header=] whose value MUST be a [=structured header/string=][[!RFC8941]]. Its
+value is the result of [=getting the platform version=] with [=user agent/platform brand=].
 
-[=User agents=] SHOULD return a version with a platform-specific format that allows differentiating
-significant platform versions by running these steps:
-    1. If the platform is Linux:
+To <dfn>get the platform version</dfn>, given |platform|, run the following steps:
```
Comment (yoavweiss) [inline index.bs:651]: Should |platform| be a string? We don't really define the type of platform brand, but a string seems appropriate.

**Review by yoavweiss — APPROVED** [review]
LGTM % question

File: index.bs (hunk 1 above)
Comment (miketaylr) [reply index.bs:651]: Yeah, probably. Not sure any other type makes sense anyways.

**Pull request merged**